### DISCUSSION TITLE
How can I upload multiple images for a listing with just a single API call?

### DISCUSSION THREAD

**Discussion by xnoname79** [etsy/open-api#1109] [Q&A]
Hi Team,

I want to upload multiple images (up to 10 images) for a listing with just one API call. It would be helpful if we had API support for this feature. It would be even more helpful if we could upload the images using the same listing creation API.

I also tried to upload multiple images by appending files to the 'image' field in form data format, but it only takes the last image to be uploaded.

This is the request body I created in Postman
![Screenshot from 2023-09-11 13-07-01](https://github.com/etsy/open-api/assets/25563537/cbcb1a2b-5ac2-4f38-8c63-21b822ecba69)

Thank you in advance!

**Comment by turboturtleD** [comment]
I’m pretty sure the API already allows you to upload multiple images at once.

**Reply by xnoname79** [reply]
Hi @turboturtleD. Thank you for the response. 
Could you please share the correct request body format and the response (if possible)? 
Maybe that's something I missed in the API call.

**Reply by turboturtleD** [reply]
I was fairly confident that I tested uploading multiple images early on in Beta.  I don’t see anything while scrolling through the API reference on my phone.  I will try to look back at that and get back to you if no one else offers more definite information before then.

**Reply by xnoname79** [reply]
I see, no worries @turboturtleD. Let me know if you find anything cool.

**Reply by turboturtleD** [reply]
I dug into this deeper, and it looks like the endpoint that I was using during early API testing where I could upload multiple images no longer exists (it was: https://openapi.etsy.com/v3/application/listings/{listing_id}/images).

I suspect that the multiple image upload functionality therefore also no longer exists.
